aidial_adapter_openai/utils/tokens.py

```diff
@@ -71,10 +70,18 @@ def discard_messages(
             prompt_tokens += tokenizer.calculate_tokens_per_message(message)
 
     if max_prompt_tokens < prompt_tokens:
-        raise HTTPException(
-            message=f"The token size of system messages ({prompt_tokens}) exceeds prompt token limit ({max_prompt_tokens})",
+        raise DialException(
+            message=(
+                f"The token size of system messages ({prompt_tokens}) "
+                f"exceeds prompt token limit ({max_prompt_tokens})"
+            ),
             status_code=400,
             type="invalid_request_error",
+            display_message=(
+                f"The token size of system messages and the last user message ({prompt_tokens})"
```

Comment:
1. Let's introduce a variable to avoid repeating the same error message twice.
2. Please add a missing space before "Try".

Reply:
Done